packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -339,37 +356,46 @@ class _OutlineButtonState extends State<OutlineButton> with SingleTickerProvider
 
   // TODO(hmuller): this method is the same as FlatButton
   Color _getSplashColor(ThemeData theme, ButtonThemeData buttonTheme) {
-    if (widget.splashColor != null)
-      return widget.splashColor;
+    if (widget.splashColor != null) return widget.splashColor;
 
     switch (_getTextTheme(buttonTheme)) {
       case ButtonTextTheme.normal:
       case ButtonTextTheme.accent:
         return theme.splashColor;
       case ButtonTextTheme.primary:
         return theme.brightness == Brightness.dark
-          ? Colors.white12
-          : theme.primaryColor.withOpacity(0.12);
+            ? Colors.white12
+            : theme.primaryColor.withOpacity(0.12);
     }
     return Colors.transparent;
   }
 
+  /// The button outline has a color and a width.
+  ///
+  /// If [this.borderSize] is provided and the style is none, then that is used.
+  ///
+  /// Else, the color and width resolve to the most specific provided value,
+  /// or the first available theme value.
```

Comment:
which one? The most specific provided value, or the first available theme value?

Reply:
Updated the docs to be more clear, still need to write test